### DISCUSSION TITLE
Input delay when any capturing software is hooked into the game

### DISCUSSION THREAD

**Discussion by piee-kun** [ppy/osu#30516] [Q&A]
When either ShadowPlay or Discord hooks into the game in order to start streaming, my game starts getting noticable input delay despite being in fullscreen. This does not happen when using OBS.

**Comment by peppy** [comment]
Have you tried with both directx and opengl?

**Reply by piee-kun** [reply]
D3D - issue occurs
Vulkan - game freezes on one frame when starting stream
OGL - issue doesn't occur

**Reply by piee-kun** [reply]
just as a side note, injecting SpecialK and enabling NVIDIA Reflex for low latency + boost eliminates this on all APIs so it might be something to do with either the game forcing Vsync during capture or something?
